### PR TITLE
[patch] fix AUS SA, they have a table now hooray

### PR DESCRIPTION
**Pull request by camjc**
image

### CODE REVIEW

File: src/shared/scrapers/AUS/SA/index.js [hunk 1]
```diff
@@ -2,6 +2,20 @@ import * as parse from '../../../lib/parse.js';
 import * as fetch from '../../../lib/fetch/index.js';
 import maintainers from '../../../lib/maintainers.js';
 
+const getKey = rowLabel => {
+  const lowerLabel = rowLabel.toLowerCase();
+  if (lowerLabel.includes('confirmed cases')) {
+    return 'cases';
+  }
+  if (lowerLabel.includes('icu')) {
+    return 'hospitalized';
```
Comment (camjc) [inline src/shared/scrapers/AUS/SA/index.js]: random guess, if we have a standard name for this please let me know.

File: src/shared/scrapers/AUS/SA/index.js [hunk 2]
```diff
@@ -17,14 +31,29 @@ const scraper = {
...
+      const $ = await fetch.page(this.url);
+      const paragraph = $('.middle-column p:first-of-type').text();
+      const { casesString } = paragraph.match(/been (?<casesString>\d+) confirmed cases/).groups;
+      return {
+        state: scraper.state,
+        cases: parse.number(casesString)
+      };
+    },
+    '2020-3-27': async function() {
+      scraper.type = 'table';
```
Comment (camjc) [inline src/shared/scrapers/AUS/SA/index.js]: Is there a better way to change type per date of scraper?

File: src/shared/scrapers/AUS/SA/index.js (hunk 1 above)
Comment (lazd) [reply src/shared/scrapers/AUS/SA/index.js]: We don't, but that's a good start!

File: src/shared/scrapers/AUS/SA/index.js (hunk 2 above)
Comment (lazd) [reply src/shared/scrapers/AUS/SA/index.js]: Nah, just `this.type` (`scraper.type` works ofc, too)

**Pull request merged**